src/redisjson.rs

```diff
@@ -443,4 +444,30 @@ pub mod type_methods {
         let json = &*(value as *mut RedisJSON);
         raw::save_string(rdb, &json.data.to_string());
     }
+
```

Comment:
Maybe add a comment why its needed, otherwise someday someone will remove it again.